DesignSafe-CI/portal
# Bug/DES-2080: Search Results Direct to First Publication Version

## Overview: ##
Fix a bug where search results were directing to the first version of a publication instead of the most recent.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2080](https://jira.tacc.utexas.edu/browse/DES-2080)

## Summary of Changes: ##
Adds the revision number to the set of fields returned in search results.

## Testing Steps: ##
1. Make sure that you have a versioned publication somewhere in your local environment.
2. Search for this publication in the Data Depot.
3. Check that the search result points to a versioned url, e.g. if you hover over the link you should see something like: 
![image](https://user-images.githubusercontent.com/12601812/152013708-a6847e08-94ce-432c-9513-407cc3a7a50d.png)
